src/mixins/filing-template-mixin.ts

```diff
@@ -530,18 +546,39 @@ export default class FilingTemplateMixin extends DateMixin {
     this.setFoundingDate(draftFiling.business.foundingDate)
 
     // restore Restoration data
+    if (draftFiling.restoration.applicationDate) {
+      this.setRestorationApplicationDate(draftFiling.restoration.applicationDate)
+    }
+    this.setRestorationApprovalType(draftFiling.restoration.approvalType)
+    if (draftFiling.restoration.courtOrder) {
+      this.setRestorationCourtOrder(draftFiling.restoration.courtOrder)
+    }
     this.setRestorationType(draftFiling.restoration.type)
-    this.setRestorationExpiry(draftFiling.restoration.expiry || null)
-    this.setRestorationRelationships(draftFiling.restoration.relationships || [])
+    if (draftFiling.restoration.expiry) {
+      this.setRestorationExpiry(draftFiling.restoration.expiry)
```

Comment:
Is there a benefit for this rather than doing ```this.setRestorationExpiry(draftFiling.restoration.expiry || null)``` as before?

Reply:
Yes -- we don't need to set a default value here. It's already set when the store state is initialized. (DRY)

Reply:
@JazzarKarim OK to resolve this conversation?